docs/layers/ecs/tutorials/1password-scim-bridge.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: "Deploy 1Password SCIM Bridge"
+sidebar_label: "1Password SCIM Bridge"
+description: "Deploy the 1Password SCIM Bridge for ECS environments"
+---
+
+import Intro from "@site/src/components/Intro";
+import Steps from "@site/src/components/Steps";
+import Step from "@site/src/components/Step";
+import StepNumber from "@site/src/components/StepNumber";
+import CollapsibleText from "@site/src/components/CollapsibleText";
+
+<Intro>
+  The 1Password SCIM Bridge is a service that allows you to automate the management of users and groups in 1Password. This guide will walk you through deploying the SCIM Bridge for ECS environments.
+</Intro>
+
+## Implementation
+
+The implementation of this is fairly simple. We will generate credentials for the SCIM bridge in 1Password, store them in AWS SSM Parameter Store, and deploy the SCIM bridge ECS service.
+
+<Steps>
+  <Step>
+    ### <StepNumber/> Generate Credentials for your SCIM bridge in 1Password
+
+    The first step is to generate credentials for your SCIM bridge in 1Password. We will pass these credentials to Terraform and the ECS task definition to create the SCIM bridge.
+
+    <Steps>
+      1. Log in to your 1Password account
+      1. Click Integrations in the sidebar.
+      1. Choose your identity provider from the User Provisioning section.
+      1. Follow the onscreen instructions to generate credentials for your SCIM bridge.
+      1. Store the credentials in AWS SSM Parameter Store.
+
+        <Steps>
```

Comment:
I believe the intendion needs to start after the `1. ` and it is currently at `1.`. I would need to look at the HTML to see if it properly nests the list underneath the previous bullet or not.

Reply:
this is what it looks like. I'm not sure why the preview env isnt working, but this is local:
![CleanShot 2024-08-20 at 12 59 22@2x](https://github.com/user-attachments/assets/3d73108a-bfca-4fe4-80c8-b6107c3b7fe6)

Reply:
fixed indentation